setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import find_packages
+from setuptools import setup
+from pip._internal.req import parse_requirements
+import os
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+
+# Pulls pip packages with versions from the requirements file
+install_requires = parse_requirements("requirements.txt", session="snowchange")
+test_requires = parse_requirements("requirements.txt", session="snowchnage")
+
+setup(
+    name="snowchange",
+    version="0.0.2",
+    author="jamesweakley/jeremiahhansen/zeitgeistf",
+    author_email="shawnsfeng@gmail.com",
+    description="Python package for managing Snowflake schema.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/zeitgeistf/snowchange",
```

Comment:
Can this point to the official project repo instead? https://github.com/jamesweakley/snowchange

Reply:
Updated